src/MoBi.Presentation/Presenter/EditParametersInContainerPresenter.cs

```diff
@@ -182,6 +182,7 @@ private void createParameterCache(IReadOnlyList<IParameter> parametersToEdit, IR
 
          initializeIndividualParameterCache(individualParameters, selectedIndividual);
          _allParametersDTO.AddRange(_individualParameterCache.Keys);
+         _allParametersDTO.Sort((a,b) => string.Compare(a.Name, b.Name, StringComparison.InvariantCultureIgnoreCase));
```

Comment:
I added this to sort the list after it's created from two different sources. Previously the individual stuff was always after the container stuff.